semeio/workflows/localisation/local_config_script.py

```diff
@@ -84,27 +86,35 @@ def run(self, *args):
  - Specify to automatically run the workflow after the initial ensemble is created,
    but before the first update by using the HOOK_WORKFLOW keyword
    with the option PRE_FIRST_UPDATE.
+ - To QC the specification of the config file for localisation, it is possible to
+   run the workflow before running initial ensemble also, but due to limitations
+   in ERT implementation, GEN_PARAM type of parameter nodes will have empty
+   list of parameters if the workflow is run before initialization. If  GEN_PARAM
+   nodes are used in correlation groups, an error message may appear in this case.
```

Comment:
`an error message may appear in this case` perhaps add the full sentence of when the error message will appear instead of saying `may`?

Reply:
I will try to reformulate something here.